utilities/batch_generate_apis.py

```diff
@@ -73,13 +74,27 @@ def generate(artman_yaml):
     generate('google/cloud/websecurityscanner/artman_websecurityscanner_v1alpha.yaml')
 
 
+def run_discogapic_gen(discovery_repo):
+    def generate(artman_yaml):
+        # Run java_discogapic task. No proto or grpc libraries are generated.
+        generate_api.run_generate_api(os.path.join(discovery_repo, artman_yaml),
+                                      "java_discogapic",
+                                      root_dir=discovery_repo,
+                                      has_proto_libs=False)
```

Comment:
I tried to make the argument list for running generation as minimal as possible. To match what I did for gapic generation, you would have to:

1. Automatically figure out the root dir from the yaml file path. For java_gapic, I search for `/google/`.
2. Automatically figure out there are no proto libs. You can probably just figure this out based on the fact that you are using "java_discogapic". 

We would probably go even further and detect whether a generation is for gapic or discogapic based on whether the full path has `/googleapis` or `/discovery` in it.

Reply:
Ok, removed all extra params except for artifact_type.

I'm a little wary of logic that is tied to the existence of the api repo name somewhere in full path, and I'd rather the artifact_type be explicit.